<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weighted Path</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,3">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,1,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1,2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetFixedSize</enum>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="start">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>그리기</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="reset">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>맵초기화</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>날짜</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QComboBox" name="year">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>2016</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2017</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2018</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2019</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2020</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2021</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2022</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2023</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="month">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>01</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>02</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>03</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>04</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>05</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>06</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>07</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>08</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>09</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>11</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>12</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="date">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>01</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>02</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>03</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>04</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>05</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>06</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>07</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>08</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>09</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>11</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>12</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>13</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>14</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>15</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>17</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>18</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>19</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>20</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>21</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>22</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>23</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>24</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>25</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>26</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>27</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>28</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>29</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>30</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>31</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="random">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Random</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>항로선택</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="zmap">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="clabel0">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>0번항로(빨강)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="corse0">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="clabel1">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>1번항로(초록)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="corse1">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="clabel2">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>2번항로(노랑)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="corse2">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="clabel3">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>3번항로(파랑)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="corse3">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>4번항로(보라)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="corse4">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
        <item>
         <widget class="QTextEdit" name="outputconsole">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView_2"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="logo"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
